infra-gen2/infra-common/src/auth-user-extensions/auth-user-extensions.ts

```diff
@@ -23,6 +24,7 @@ export const addAuthUserExtensions = ({
   addCustomSenderLambda({ name, stack, cfnUserPool, graphQL });
   addCreateUserLambda({ name, stack, userPool, graphQL });
   addDeleteUserLambda({ name, stack, userPool, graphQL });
+  addEnableSmsMfaLambda({ name, stack, userPool, graphQL });
```

Comment:
Q: Will this add the lambda to every backend that calls this method? Can this be optional?

Reply:
Yes it will add it to each env. It could be optional. So could the other lambdas. I think it is simpler to add all the lambdas to each env rather than try to remember which envs have which lambda. It doesn't really cost anything if it doesn't get invoked.

Reply:
It doesn't need to come from this PR, but it would be a nice optimization to have these be pulled in as needed. Besides reducing the amount of resources added to the AWS account, it may speed up deployment of the entire infra stack. May also prevent maintenance confusion by having the bare minimum for each env.